### ISSUE TITLE
Can't load up file after closing it

### ISSUE THREAD

**Issue by HalloM8** [processing/processing#5073]
Once I closed the program, I could open the source code again by double clicking the project file. However, I can't run it. All of the buttons on the top seem to disappear, and once I click run, there are a ton of error messages in the console. I tried opening Processing by itself but nothing shows up. I also tried to copy all of the code and put it on a Java file but that didn't seem to work either. I have checked the code thoroughly and nothing seemed to bother me like a missing semicolon. In the end, I managed to get the file working by uninstalling Processing and installing it again. However, this only worked once and the bug seemed to happen again. I have also tried installing Processing 3, and the error messages still showed up. Please tell me what's gone wrong...

[Epileptic_Checkered_Floor.zip](https://github.com/processing/processing/files/1011900/Epileptic_Checkered_Floor.zip)

**Comment by JakubValtar** [comment]
@HalloM8 Could you please share with us what the error messages say? Just copy paste them here or make a screenshot if you can't select the text.

**Comment by HalloM8** [comment]
![screen shot 2017-05-19 at 19 25 53](https://cloud.githubusercontent.com/assets/28789455/26261514/5c4307ba-3cc9-11e7-9715-127d602ed5d1.png)
![screen shot 2017-05-19 at 19 25 16](https://cloud.githubusercontent.com/assets/28789455/26261515/5f815b52-3cc9-11e7-9c8c-6767a735b637.png)

**Comment by JakubValtar** [comment]
@benfry Any ideas? Looks like some problem with installation/Java mode?

**Comment by HalloM8** [comment]
Clueless?

**Comment by jeremydouglass** [comment]
Long shot, but if you reinstalled Pricessing and then it worked fine *once* my guess would be that your sketch is doing something destructive to the Processing install itself. Are you saving files or deleting files with your sketch, and perhaps you are saving them inside the PDE app path space?

**Comment by benfry** [comment]
Yeah, appears to be missing files. Which can sometimes be caused by permissions issues. But one of those errors is the .html file for the Welcome screen going missing… I haven't a clue how you'd make that happen, except for a sketch manipulating files inside Processing.app, or some sort of overly aggressive virus scanning software locking up files.

**Comment by JakubValtar** [comment]
@benfry Could this happen when you install a new version of Processing while the old version is still running in the top bar?

**Comment by benfry** [comment]
Not sure; shouldn't be possible to run more than one copy at a time.
